tests/integ/modin/test_modin_stored_procedures.py

```diff
@@ -3,27 +3,51 @@
 # Copyright (c) 2012-2024 Snowflake Computing Inc. All rights reserved.
 #
 
+import os
+
 import modin.pandas as pd
+import pytest
 
 from snowflake.snowpark import Session
 from snowflake.snowpark.functions import sproc
 from snowflake.snowpark.modin.plugin import (
+    actual_pandas_version,
     supported_modin_version,
-    supported_pandas_version,
 )
 from tests.integ.utils.sql_counter import sql_count_checker
 
 PACKAGE_LIST = [
-    f"pandas=={supported_pandas_version}",
+    # modin 0.30.1 supports any pandas 2.2.x, so just pick whichever one is installed in the client
+    f"pandas=={actual_pandas_version}",
```

Comment:
do we need to import that from snowflake.snowpark.modin.plugin? can we just get the current pandas version here?

Reply:
that would work as well, do you have a preference for either way?